msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -241,6 +241,73 @@ The following code is an example of a Response rendering template: <br/>
 |`responseCardTemplate`    |  `adaptiveCardTemplate`  | A template for creating an Adaptive Card from a result entry.      |   Yes      |
 |`previewCardTemplate`     |  `previewCardTemplate`       | A template for creating a preview card from a result entry. The resulting preview card is displayed in the message extension flyout menu.        |  Yes       |
 
+##### Multi parameters
```

Comment:
Can you move this down, or maybe into its own document? Introducing the concept of multiple parameters this early is likely to confuse the reader.

Consider explaining the single-parameter case first, since that maps directly to the query-based command concept. Then introduce the ability to add more parameters, and starting by explaining the use cases and APIs where they are needed.

Reply:
Updated in article [Build API-based message extension](https://review.learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/build-api-based-message-extension?branch=pr-en-us-10532&tabs=token-v2#multi-parameters).